be/test/common/config_test.cpp

```diff
@@ -210,4 +380,17 @@ TEST_F(ConfigTest, test_read_write_mutable_string_concurrently) {
     }
 }
 
+TEST_F(ConfigTest, test_alias) {
+    CONF_mInt16(cfg_int32, "8000");
+    CONF_Alias(cfg_int32, cfg_int32_alias);
+
+    std::stringstream ss;
+    ss << R"DEL(
+        cfg_int32_alias = 8080
```

Comment:
try to define multiple aliases to the same CONF item, they should all work.

Reply:
Updated, PTAL